security/fma-isthmus-l2-pectra.md

```diff
@@ -0,0 +1,207 @@
+
+# Pectra Features on Isthmus
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Failure Modes and Recovery Paths](#failure-modes-and-recovery-paths)
+  - [FM1: Chain halt due to upgrade transaction failing](#fm1-chain-halt-due-to-upgrade-transaction-failing)
+  - [FM2: Chain halt due to request hash mismatch](#fm2-chain-halt-due-to-request-hash-mismatch)
+  - [FM3: EIP-7702 transactions cannot be included on the chain](#fm3-eip-7702-transactions-cannot-be-included-on-the-chain)
+  - [FM4: BLS Precompiles could cause increased FP Program execution time](#fm4-bls-precompiles-could-cause-increased-fp-program-execution-time)
+  - [FM5: BLS Precompiles could fail to execute in the FP program](#fm5-bls-precompiles-could-fail-to-execute-in-the-fp-program)
+  - [FM6: Increased call data cost affects network upgrade transactions](#fm6-increased-call-data-cost-affects-network-upgrade-transactions)
+  - [FM7: Early fork if batches containing EIP-7702 transactions could be posted before Pectra](#fm7-early-fork-if-batches-containing-eip-7702-transactions-could-be-posted-before-pectra)
+  - [FM8: Smart contracts relying on sender check no longer ensures sender does not have code](#fm8-smart-contracts-relying-on-sender-check-no-longer-ensures-sender-does-not-have-code)
+  - [Generic items we need to take into account:](#generic-items-we-need-to-take-into-account)
+- [Action Items](#action-items)
+- [Audit Requirements](#audit-requirements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+<!-- _Italics are used to indicate things that need to be replaced._ -->
+
+|                    |                                                    |
+| ------------------ | -------------------------------------------------- |
+| Author             | Julian Meyer                                       |
+| Created at         | 2025-02-10                                         |
+| Initial Reviewers  | refcell                                            |
+| Need Approval From |                                                    |
+| Status             | In Review 🔎                                       |
```

Comment:
We need to add this add the end of the current review please not accept this current suggestion now. 
I will accept this at the end myself. 
```suggestion
| Status             |  Implementing Action 🛫                                       |
```